Prog/Preservation_database.sas

```diff
@@ -560,4 +608,43 @@ if NHPDPropertyID = "95fadb37-c2ff-e611-8115-74d435edc0c2" then do;
 	End;
 run;
 
+proc contents data=test;
+run;
+
+/*create labels for variables*/
+
+proc format;
+	value COG
+    1= "COG county"
+    0="Non COG county";
+
+run;
+
+/*create COG region flag*/
+data all;
+set test;
+  if CountyCode in ("11001", "24017", "24021", "24031", "24033", "51013", "51059", "51107", "51153", "51510", "51600", "51610", "51683", "51685") then COGregion =1;
+  else COGregion=0;
+  format COGregion COG. ;
+run;
+
+/*assign label to variables*/
+data all;
+set all;
+  format  ;
+run;
+
+/*finalize dataset*/
+
+%Finalize_data_set(
+/** Finalize data set parameters **/
+data=all,
+out=all_finalize,
+outlib=RegHsg,
+label="Compile preservation data for region with COG flag",
```

Comment:
More like "National Preservation Database Active and Inconclusive Properties 1/2019"